lib/tasks/seed.rake

```diff
@@ -0,0 +1,95 @@
+require 'factory_girl'
+
+namespace :seed do
```

Comment:
Tasks like `rake seed:all` & `rake seed:hosts` seems a little generic to me, I might be unsure want I'm really seeding. What about using a different namespace, maybe `seed:forgeries` ?